### DISCUSSION TITLE
retrieve env vars for a project API fails

### DISCUSSION THREAD

**Discussion by JigarJoshi** [vercel/community#2508] [Bug Report]
```
curl --location --request GET 'https://api.vercel.com/v9/projects/<prj_id>' \
--header 'Authorization: bearer <token>'
```

Returns the project with all its env variables. 

But doing the same to retrieve all of its env vars, returns empty env vars

```
curl --location --request GET 'https://api.vercel.com/v9/projects/<prj_id>/env' \
--header 'Authorization: bearer <token>'
```

returns

```
{
    "envs": []
}
```
with 200 OK.

Note: this behavior is for certain project. This project belongs to personal account (no team id).

**Comment by amyegan** [comment]
Hi @JigarJoshi. It sounds like this isn't happening on all projects, but only a specific one. If that's correct, would you mind sharing the project id so we can look into it?
